test/src/main/java/org/apache/accumulo/test/server/security/SystemCredentialsIT.java

```diff
@@ -86,7 +87,7 @@ public static void main(final String[] args) throws AccumuloException, TableNotF
         client.securityOperations().authenticateUser(creds.getPrincipal(), creds.getToken());
         try (Scanner scan = client.createScanner(RootTable.NAME, Authorizations.EMPTY)) {
           for (Entry<Key,Value> e : scan) {
-            e.hashCode();
+            assertNotNull(e.hashCode());
```

Comment:
assertNotNull would force an auto-boxing of the hashCode int. That part's probably okay, since we don't actually care about the value of the hash code. However, I'd probably try to avoid a JUnit assert call inside the main method. I'm not sure how the test would behave if the main method ended up throwing an AssertionError instead of the expected System.exit from the below checks.

Reply:
I reworked to replace the hashCode() call with an iterator.next(). Think that should serve similar purpose. @ctubbsii  when you have a moment can you take another look at the changes I have made and see if they address your comments.